model/book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
```

Comment:
- [ ] Please kindly implement the method to adding rentals as listed in the project requirement. More so the rentals variable supposed to be an `attr_accessor` because of the many to many relationship between  **Person** and **Book** using the intermediate class **Rental**.